Postal/Postal.swift

```diff
@@ -268,7 +268,7 @@ public extension Postal {
 
 private extension Postal {
     
-    func doAsync<T, E: Error>(_ f: @escaping () throws -> T, completion: @escaping (Result<T, E>) -> Void) {
+    func doAsync<T, E>(_ f: @escaping () throws -> T, completion: @escaping (Result<T, E>) -> Void) {
```

Comment:
was it intended ?

Reply:
It's to fix a warning `Redundant conformance constraint 'E': 'Error'`.

Reply:
Because result already constraints `E: Error` ? Interesting, did not know that feature of swift 4. Thank you !

Reply:
Me neither, was surprised actually.

Reply:
What's surprising is that it did not trigger in some other places, but we did not use a lot of generic constraints in Postal